apps/bfd-server/dev/api-changelog.md

```diff
@@ -1,5 +1,16 @@
 # API Changelog
 
+## BFD-1917: Map FI Claim Process Date in V1 and V2
+
+FI Claim Process Date data existed in our code but was not mapped for V1 or V2. This
+data is now available in the extensions for SNF, HHA, Hospice, Inpatient and Outpatient claims:
+```json
+    {
+      "url" : "https://bluebutton.cms.gov/resources/variables/fi_clm_proc_dt",
+      "valueDate" : "2014-02-07"
+    }
+```
```

Comment:
Similar to my comment on the other PR, we may want to include more structural context in the snippet by providing the snippet from the top level (with ellipsis for skipped content like in other changelog examples), and perhaps also provide the path to this change (eob.extension[N].valueDate)

Reply:
Updated the json for this.